src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.storage.StorageList;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Delete Command to remove the tasks users do not want, according to its number on the list.
+ */
+public class DeleteCommand extends Command {
+    private int listNum;
```

Comment:
Can I clarify what `listNum` is used for? Would it be better to call it `index` or something?